demisto_sdk/commands/unify/unifier.py

```diff
@@ -216,8 +216,17 @@ def get_code_file(self, script_type):
         if self.package_path.endswith('ApiModule'):
             return os.path.join(self.package_path, os.path.basename(os.path.normpath(self.package_path)) + '.py')
 
+        yml_name = os.path.basename(self.yml_path).split('.')[0]
+
         script_path = list(filter(lambda x: not re.search(ignore_regex, x),
-                                  glob.glob(os.path.join(self.package_path, '*' + script_type))))[0]
+                                  glob.glob(os.path.join(self.package_path, f'{yml_name}' + script_type))))
```

Comment:
why do we need `glob` if you search for a file with a specific name?
just use `script_file_path = os.path.splitent(self.yml_path)[0] + '.py'`